resources/configure_dependencies.py

```diff
@@ -465,14 +465,14 @@ def InstallPETSC(pkg: str, ver: str, gold_file: str):
             log_file.write(f"{command}\n{err}\n")
             package_log_file.write(f"{command}\n{err}\n")
 
-        command = f"make all -j{argv.jobs}"
+        command = f"make OMAKE_PRINTDIR=gmake all -j{argv.jobs}"
         success, err = ExecSub(command, out_log=package_log_file, env_vars=env_vars)
         if not success:
             print(command, err)
             log_file.write(f"{command}\n{err}\n")
             package_log_file.write(f"{command}\n{err}\n")
 
-        command = "make install"
+        command = "make OMAKE_PRINTDIR=gmake install"
```

Comment:
```suggestion
        command = "make install"
```
The fix is only needed for `make all`. If you had trouble without it here as well, feel free to disregard. I have never used this here and have never had trouble

Reply:
I followed your suggestion. Then I did a ./configure clean and ./configure. Then re-compile and passed all tests. If this is good enough, then we can go with your suggestion. I think I did a replace-all, that's why line 475 was changed as well.

Reply:
Just an FYI. The configure.sh only operates on the chi_build directory. Does nothing to dependencies. To really test it you should clean out your dependencies directory and use resources/configure_dependencies.py again.